### ISSUE TITLE
Add windows guidelines in readme

### ISSUE THREAD

**Issue by GouravSardana** [IndianOpenSourceFoundation/dynamic-cli#123]
Add the guidelines to install dynamic on the window OS because the user needs to set up WSL in the window OS to make it work. We are working on it to make it independent but some of the binaries are different in window and termios is causing the issue. For more detail refer #94

**Comment by pranavbaburaj** [comment]
I think it will be better to just link a guide on installing wsl rather than writing up one on our own. What do you think?

**Comment by gourav-bhnetwork** [comment]
Yup. Adding a link would be better. Giving an example screenshot would work. Maybe we can make a new file called window_setup.MD and add the steps + links + screenshot

**Comment by pranavbaburaj** [comment]
Hmmm yeah.

**Comment by saidhanya18** [comment]
could you please assign this issue to me

**Comment by saidhanya18** [comment]
![image](https://user-images.githubusercontent.com/94187364/156913943-fae87493-69b3-40be-97b9-49dc82d82612.png)

**Comment by GouravSardana** [comment]
You can’t push directly to our repository. Process is -> fork the repo in your account + make change+ push the code to the forked repo + make a PR(Pull request)

**Comment by saidhanya18** [comment]
> 

okay thank you
